docs/eventing/brokers/broker-developer-config-options.md

```diff
@@ -8,31 +8,31 @@ The following is a full example of a multi-tenant (MT) channel-based Broker obje
 apiVersion: eventing.knative.dev/v1
 kind: Broker
 metadata:
-   name: default
-   namespace: default
-   annotations:
-     eventing.knative.dev/broker.class: MTChannelBasedBroker
- spec:
-   config:
-     apiVersion: v1
-     kind: ConfigMap
-     name: config-br-default-channel
-     namespace: knative-eventing
-   delivery:
-     deadLetterSink:
-       ref:
-         kind: Service
-         namespace: example-namespace
-         name: example-service
-         apiVersion: v1
-       uri: example-uri
-     retry: 5
-     backoffPolicy: exponential
-     backoffDelay: "2007-03-01T13:00:00Z/P1Y2M10DT2H30M"
+  name: default
+  namespace: default
+  annotations:
+    eventing.knative.dev/broker.class: MTChannelBasedBroker
+spec:
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: config-br-default-channel
+    namespace: knative-eventing
+  delivery:
+    deadLetterSink:
+      ref:
+        kind: Service
+        namespace: example-namespace
+        name: example-service
+        apiVersion: v1
+      uri: example-uri
+    retry: 5
+    backoffPolicy: exponential
+    backoffDelay: "2007-03-01T13:00:00Z/P1Y2M10DT2H30M"
 ```
 
 - You can specify any valid `name` for your broker. Using `default` will create a broker named `default`.
-- The `namespace` must be an existing namespace in your cluster. Using `default` will create the broker in the current namespace.
-- You can set the `eventing.knative.dev/broker.class` annotation to change the class of the broker. The default broker class is `MTChannelBasedBroker`, but Knative also supports use of the `Kafka` broker class. For more information about Kafka brokers, see the [Apache Kafka Broker](../brokers/broker-types/kafka-broker/README.md) documentation.
+- The `namespace` must be an existing namespace in your cluster. Using `default` will create the broker in the `default` namespace.
```

Comment:
Is this correct re the broker namespace?
@knative/eventing-wg-leads

Reply:
This is correct. Here `default` is just an example.